service/raft/raft_address_map.hh

```diff
@@ -205,6 +205,13 @@ class raft_address_map_t : public peering_sharded_service<raft_address_map_t<Clo
         });
     }
 
+    void replicate_add_or_update_entry(const raft::server_id& id, const gms::inet_address& ip_addr,
```

Comment:
I thought we agreed that the caller should be responsible for setting mappings on the different shards...

Reply:
Discussed and resolved we have to keep it.